cross/duplicity/Makefile

```diff
@@ -1,15 +1,15 @@
 PKG_NAME = duplicity
-PKG_SHORT_VERS = 0.6
-PKG_VERS = $(PKG_SHORT_VERS).22
+PKG_SHORT_VERS = 0.7
+PKG_VERS = $(PKG_SHORT_VERS).03
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://launchpad.net/$(PKG_NAME)/$(PKG_SHORT_VERS)-series/$(PKG_VERS)/+download
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/librsync
+DEPENDS = cross/boto cross/librsync
```

Comment:
`cross/boto` is a wheel. So, it belongs in `spk/duplicity/Makefile` as `BUILD_DEPENDS`, not as a dependency of another wheel.